misc/import_stats.sql

```diff
@@ -13,6 +13,7 @@ CREATE TABLE `get5_stats_matches`
     `team1_score` smallint(5) unsigned NOT NULL DEFAULT '0',
     `team2_name`  varchar(64)          NOT NULL DEFAULT '',
     `team2_score` smallint(5) unsigned NOT NULL DEFAULT '0',
+    `server_id`   smallint(5) unsigned NOT NULL DEFAULT '0',
```

Comment:
I would have used a `int`, and maybe `int(10) unsigned` as for the matchid.

Reply:
You're right. smallint is probably too small. I'll change it.